src/applications/BreastTexturePipeline/src/BreastTexturePipeline.cxx

```diff
@@ -224,14 +218,28 @@ int main(int argc, char** argv)
   //case 2:
   //{
     //using ImageType = itk::Image< float, 2 >;
-  return algorithmsRunner();
 
+  cbica::createDir(loggerFolder);
+  cbica::createDir(downloadFolder);
+
+  ApplicationDownloadManager* appDownloadMngr = new ApplicationDownloadManager();
```

Comment:
Please replace
```cpp
ApplicationDownloadManager* appDownloadMngr = new ApplicationDownloadManager();
```
with
```cpp
ApplicationDownloadManager appDownloadMngr;
```
This would remove the requirement for us to check for memory leaks. You might need to change the way this object gets connected afterwards.

Reply:
Got it